keyboards/playkbtw/helen80/readme.md

```diff
@@ -0,0 +1,15 @@
+# Helen 80
+
+![Helen 80](https://cdn.store-assets.com/s/409567/i/18788640.png)
+
+A 80% RGB hotswap keyboard kit made by Play Keyboard.
+
+* Keyboard Maintainer: [Barry Huang](https://github.com/yj7272098)
+* Hardware Supported: Helen 80
+* Hardware Availability: [Play Keyboard](http://play-keyboard.store/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make playkbtw/helen80:default
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?